### DISCUSSION TITLE
how many requests to get banned

### DISCUSSION THREAD

**Discussion by torgodly** [nguyenary/QRCodeMonkey#13] [Q&A] [answered]
hello, I want to use this package in a project that is gonna make a lot of requests to the monkey API. I wanted to ask how many requests until the server gets banned and if is it reliable or if I should look for another package...

thank you :)

**Comment by nguyenary** [comment, marked as answer]
If you need to handle many requests for your project. I think you should find more reliable packagist :D

**Reply by torgodly** [reply]
any suggestions?
